src/airmobility/object_a.proto

```diff
@@ -186,8 +189,13 @@ message VoxelValueForecastWeather {
 
 message ObjectMobileNetwork { // 電波強度オブジェクトの値。空間IDを含む。
     optional string reference = 1 ; // 素となる情報へのリンク
-    string plmn_id = 2 ; // 通信事業者コード
```

Comment:
The PLMN is conventionally formatted as "001-01", but this is informal.
For the specification purpose, I would like to parse the PLMN into the MCC and MNC.